src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileParser.cs

```diff
@@ -32,8 +32,6 @@ public XamlFileDefinition[] ParseFiles(string[] xamlSourceFiles)
 			return xamlSourceFiles
 				.AsParallel()
 				.Select(ParseFile)
-				.Trim()
-                .OrderBy(f => f.UniqueID)
```

Comment:
The ordering was added to ensure the stability of the generation (w.r.t the parallel above) and provide repeatable builds.

Reply:
It's still ordered immediately after `ParseFiles()` is called, I needed the unordered files list additionally.